lib/l2meter/emitter.rb

```diff
@@ -197,10 +197,10 @@ def build_token(key, value)
     end
 
     def format_token(key, value)
-      case value
-      when TrueClass
+      case
```

Comment:
What do you think about changing this to an `if`/`elsif`/`else` rather than an empty `case` statement?

Reply:
Well... I do prefer this way because it means the conditions are aligned vertically. I think I also find it easier to parse in my head, but obviously that's quite subjective. Is there a good reason to change it?

Reply:
> Is there a good reason to change it?

Not really; personal style, I suppose. And I'm like a [3/10 on the sliding scale](https://capwatkins.com/blog/the-sliding-scale-of-giving-a-fuck) on this one, so perfectly happy to leave it.